test/test_client.py

```diff
@@ -8,6 +8,64 @@
 from radiant_mlhub.exceptions import EntityDoesNotExist, MLHubException, AuthenticationError
 
 
+
+class TestCustomUrl:
+
+    def test_custom_url_list_datasets(self, monkeypatch, requests_mock):
+        # Set up custom URL
+        custom_root_url = "https://staging.api.radiant.earth"
+        monkeypatch.setenv('MLHUB_ROOT_URL', custom_root_url)
+
+        # Mock this using requests-mock
+        url = 'https://staging.api.radiant.earth/datasets?key=test_key'
+        requests_mock.get(url, status_code=200, json=[])
+
+        # Making request to API
+        radiant_mlhub.client.list_datasets()
+
+        # Get request history and check that request was made to custom URL
+        history = requests_mock.request_history
+        assert len(history) == 1
+        assert history[0].url == "https://staging.api.radiant.earth/datasets?key=test_key"
+
+
+    def test_custom_url_get_collection(self, monkeypatch, requests_mock):
+        # Set up custom URL
+        custom_root_url = "https://staging.api.radiant.earth"
+        monkeypatch.setenv('MLHUB_ROOT_URL', custom_root_url)
+
+        # Mock this using requests-mock
+        url = 'https://staging.api.radiant.earth/datasets?key=test_key'
+        requests_mock.get(url, status_code=200, json=[])
+
+        # Making request to API
+        radiant_mlhub.client.get_collection("collection_id")
+
+        # Get request history and check that request was made to custom URL
+        history = requests_mock.request_history
+        assert len(history) == 1
+        assert history[0].url == "https://staging.api.radiant.earth/datasets?key=test_key"
+
+
+    def test_custom_url_list_collection_item(self, monkeypatch, requests_mock):
+        # Set up custom URL
+        custom_root_url = "https://staging.api.radiant.earth"
+        monkeypatch.setenv('MLHUB_ROOT_URL', custom_root_url)
+
+        # Mock this using requests-mock
+        url = 'https://staging.api.radiant.earth/datasets?key=test_key'
+        requests_mock.get(url, status_code=200, json=[])
+
+        # Making request to API
+        radiant_mlhub.client.list_collection_item()
```

Comment:
```suggestion
        _ = list(radiant_mlhub.client.list_collection_items("collection_id"))
```

Looks like there was a typo in the name of the method and you also need to pass it a collection ID as an argument. This method returns a [Python generator](https://realpython.com/introduction-to-python-generators/), so it doesn't actually make a call to the API until you loop over that generator. This suggested change does that by creating a `list` of the results.